gradle/gradle-mvn-push.gradle

```diff
@@ -1,66 +0,0 @@
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: "com.jfrog.bintray"
```

Comment:
This plugin can go

Reply:
What about the plugin above called `android-maven` ?